src/main/java/Launcher.java

```diff
@@ -0,0 +1,8 @@
+import javafx.application.Application;
+
```

Comment:
please consider adding javadoc comments to this class

Reply:
I was considering to remove this class. After all, it's a Launcher that does nothing.